### PR TITLE
Doc site p1

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created pages for the rest of the components.

### CODE REVIEW

File: css/badge.css [hunk 1]
```diff
@@ -41,9 +41,10 @@
 
 .badge-container .avatar-xs ~ .status-badge {
   position: relative;
-  height: 0.4rem;
-  width: 0.4rem;
-  right: 0.7rem;
+  height: 0.5rem;
+  width: 0.5rem;
+  right: 0.5rem;
+  bottom: 0.1rem;
```
Comment (ritikaawasthy) [inline css/badge.css:47]: use px for smaller units

**Review by ritikaawasthy — APPROVED** [review]
looks good to merge

File: css/modal.css [hunk 2]
```diff
@@ -3,8 +3,11 @@
   position: fixed;
   z-index: 9999;
   top: 0;
+  left: 0;
   width: 100%;
   height: 100%;
+  /* width: 100vh;
```
Comment (VedanthB) [inline css/modal.css:9]: remove commented code here

File: css/snackbar.css [hunk 3]
```diff
@@ -1,12 +1,13 @@
 .snackbar {
+  z-index: 99999;
```
Comment (VedanthB) [inline css/snackbar.css:2]: use sensible values for z-index, will be easier to debug

File: pages/docs.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 100%;
+  border: 1px solid var(--grey-200);
+}
+
+th,
+td {
+  text-align: left;
+  padding: 16px;
```
Comment (VedanthB) [inline pages/docs.css:37]: use rem here

File: pages/docs.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  width: 3px;
+}
+
+body::-webkit-scrollbar-track {
+  background-color: var(--indigo-500);
+}
+
+body::-webkit-scrollbar-thumb {
+  background: var(--indigo-200);
+  border-radius: 25px;
```
Comment (VedanthB) [inline pages/docs.css:24]: same here use rem here

**Review by VedanthB — APPROVED** [review]
Good work Ananya, 
Please keep your PRs short, it make it easier to get your Pr reviewed and help thr reviewer review better.
Everything else looks good to go 🚀

**Pull request merged**